app/views/change_logs/_banner.html.slim

```diff
@@ -1,4 +1,4 @@
-.moj-banner role="region" aria-label="information-#{change_log[:key]}" class=("moj-banner--warning" if change_log[:problem])
```

Comment:
ok - so here we are no longer turning the banner red as it's dealt with by `app/views/start/_banners.html.slim`